gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Dec 28 10:00:20 PST 2015
+#Wed Apr 12 18:16:54 CDT 2017
```

Comment:
I  am guessing when I updated the gradle.build suport versions and it synced it changed something